openedx/features/edly/message_types.py

```diff
@@ -0,0 +1,15 @@
+from openedx.core.djangoapps.ace_common.message import BaseMessageType
+
+
+class OutlineChangesNotification(BaseMessageType):
```

Comment:
nit: `OutlineChangeNotification`

It is generally recommended to use singular nouns for model naming.

Reply:
Done